scrapers/MindGeekAPI.py

```diff
@@ -0,0 +1,82 @@
+import json
+import os
+import re
+import sys
+from datetime import datetime
+
+import requests
+
+# Not necessary but why not ?
+user_agent = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:79.0) Gecko/20100101 Firefox/79.0'
+
+
+def debugPrint(t):
+    sys.stderr.write(t + "\n")
+
+
+def get_info(url):
+    r = requests.get(url)
+    id = re.search('(.+releaseId\":\")(.+?)(\".+)',
+                   r.text, re.IGNORECASE).group(2)
+    token = re.search('(.+jwt\":\")(.+?)(\".+)',
+                      r.text, re.IGNORECASE).group(2)
+    return id, token
+
+
+fragment = json.loads(sys.stdin.read())
+# Get the id and instance token
+if fragment["url"]:
+    id, instance_token = get_info(fragment["url"])
+else:
+    print("You need to set the URL", file=sys.stderr)
+    exit(1)
+
+api_URL = 'https://site-api.project1service.com/v2/releases/{}'.format(id)
+headers = {
+    'Instance': instance_token,
+    'User-Agent': user_agent
+}
+
+# Send to the API
+r = requests.get(api_URL, headers=headers)
+try:
+    api_json = r.json().get('result')
+    api_json['url'] = fragment["url"]
+except:
+    print("An error has occurred", file=sys.stderr)
+    print(f"Request status: `{r.status_code}`", file=sys.stderr)
+    print(f"Check your MindGeekAPI.log for more details", file=sys.stderr)
+    with open("MindGeekAPI.log", 'w', encoding='utf-8') as f:
+        f.write("Instance Token: {}\n".format(instance_token))
+        f.write("Scene ID: {}\n".format(id))
+        f.write("Request:\n{}".format(r.text))
+    exit(1)
+try:
+    # Saving the JSON to a file (Write '- logJSON' below MindGeekAPI.py in MindGeekAPI.yml)
+    if sys.argv[1] == "logJSON":
+        try:
+            os.makedirs('MindGeekAPI_JSON')
+        except FileExistsError:
+            pass  # Dir already exist
+        with open("MindGeekAPI_JSON\\" + id + ".json", 'w', encoding='utf-8') as f:
```

Comment:
this only works for windows
```python
        filename = os.path.join("MindGeekAPI_JSON", id+".json")
        with open(filename, 'w', encoding='utf-8') as f:
```
the path join should be cross platform safe instead

Reply:
Forward slashes work on Windows (or have for years now at least) but this would be the smarter option.